cmd/kube-scheduler/app/server_test.go

```diff
@@ -230,6 +231,104 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "multi profiles files",
```

Comment:
Actually, we don't need this test **and** the one with one profile. Let's just leave the multiple profiles one.

Reply:
In other words, remove the test case named `component configuration`